src/main/java/com/mallowigi/icons/TintedIconsComponent.java

```diff
@@ -83,20 +82,17 @@ public void pluginUnloaded(@NotNull final IdeaPluginDescriptor pluginDescriptor,
   }
 
   private void initComponent() {
-    colorPatcher = new TintedColorPatcher();
-    SVGLoader.setColorPatcherProvider(colorPatcher);
+    SVGLoader.setColorPatcherProvider(new TintedColorPatcher(SvgLoaderHacker.collectOtherPatcher()));
 
     // Listen for changes on the settings
     connect.subscribe(LafManagerListener.TOPIC, source -> {
-      SVGLoader.setColorPatcherProvider(null);
```

Comment:
I didn't know the reason that for the patcher being cleared before hand, was it important?